shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -88,7 +88,10 @@ private static int inputTypeFromTextInputType(String inputType,
             return InputType.TYPE_CLASS_PHONE;
 
         int textType = InputType.TYPE_CLASS_TEXT;
-        if (inputType.equals("TextInputType.emailAddress"))
+        if (inputType.equals("TextInputType.multiline")) {
+            textType |= InputType.TYPE_TEXT_FLAG_MULTI_LINE;
+        }
```

Comment:
No curly brackets for one-line conditionals.

Reply:
Done.